client/layout/masterbar/logged-in.jsx

```diff
@@ -96,7 +96,7 @@ const MasterbarLoggedIn = React.createClass( {
 					className="masterbar__item-new"
 					tooltip={ this.translate( 'Create a New Post', { textOnly: true } ) }
 				>
-					{ this.translate( 'New Post' ) }
+					{ this.translate( 'Post' ) }
```

Comment:
I like how short and to the point "Post" is in English.

Is there a "context" variable we can pass to the translate function, so translators can keep this in mind? And if yes, would that alleviate your concern?

Thanks for the review!